websites/api-docs-smoke-test/shortcodes.js

```diff
@@ -0,0 +1,8 @@
+// by convention, provide additional components that are _intended_
+// to be used in MDX content but not part of the defaults provided
+// via the MDX provider
+export {
+  ApiType,
+  ApiEndpoint,
+  ApiEndpointsForResource,
+} from '@commercetools-docs/gatsby-theme-api-docs';
```

Comment:
Isn't this the exact same thing as before? 🤔 I'm not sure I fully get the reasoning behind having a `shortcodes.js` exports vs `index.js`.

Reply:
This is in the websites folder and the name is chosen to make the import statement in the MDX pages as intuitive and logical as possible.  There was no index.js in the website before.  

Alternatives from MDX author perspective in the MDX file: 
```mdx
// no local re-export:   -> verbose and hard to memorize and understand for authors. If you want to override the component locally in the website you have to touch all pages. 
import { ApiType } from "@commercetools-docs/gatsyby-theme-api-docs"

// local re-export from an index.js:  -> not sure, shortest but the end looks like a meaningless sequence of special characters. Would it work at all? 
import { ApiType } from "/"

// using a meaningful conventional name as a symbolic token ("always write this" - no need to expose internals)
import { ApiType } from "/shortcodes"
```

I personally found the last most meaningful but still memorizable.   Question of taste maybe, we can ask the authors for their opinion.

Reply:
@stmeissner  do you have an opinion?

Reply:
One is too specific, two is missing information. Option three is fine for me too.

Reply:
Thank you, so we keep it as implemented now.